cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -54,9 +58,8 @@ def test_vmware_provider_filters(appliance, provider, soft_assert):
         soft_assert(esx_platform in all_options, "ESX Platform does not exists in options")
 
 
-@pytest.mark.manual
 @pytest.mark.tier(3)
-def test_appliance_scsi_control_vmware():
+def test_appliance_scsi_control_vmware(request, appliance):
```

Comment:
@jawatts  I would like to run this test only once, irrespective of provider, what marker should I use? Asking since this test dosn't use any provider.

Reply:
Dicussed offline, will need to override the provider marker.

Reply:
@jawatts @kedark3 I think override provider marker is not the best solution here; still it will try to add vsphere provider. The basic `pytest` concept is a `global marker` will always execute for each test in the respective file. So we need proper planning.

If we like to set this test `provider independent` (test will not try to add provider) in the same file then we need to remove a provider global marker and set on the remaining test.

Reply:
@jawatts @mshriver your opinion?

Reply:
@digitronik the global pytest marker in this case is running the test 3 times, for each vpshere version. What @kedark3 wants to do here, is run it against a vpshere provider, but just once. So
```
@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, override=True)`

```

Reply:
Added that with ONE_PER_TYPE

Reply:
aaaaaaaahh I was confused with `test dosn't use any provider` actually steps also pointing same. :+1: then its ok... we can overwrite provider marker for a specific use case.